### ISSUE TITLE
Can't access Request Cart element via keyboard-only navigation

### ISSUE THREAD

**Issue by sdellis** [pulibrary/pulfalight#547]
Prevents keyboard-only users from submitting a request that they have added to the cart.

Severity: Critical

WCAG Guideline:
https://www.w3.org/WAI/WCAG21/quickref/#keyboard

**Comment by sdellis** [comment]
This is accessible if you start from the top. However, if a user clicks on the Request Cart icon (in the header or sub-description bar) the focus should be at the top of the Request Cart slide out.
